templates/zerver/realm_creation_form.html

```diff
@@ -34,6 +34,29 @@
         <label for="realm_type" class="inline-block label-title">{{ _('Organization type') }}</label>
     </div>
 
+    <div class="input-box">
+        <div class="inline-block relative">
+            <select name="realm_default_language" id="realm_default_language">
+                {% if form.realm_default_language.value() %}
+                    {% for language in language_list %}
+                        <option value="{{ language.code }}" {% if form.realm_default_language.value() == language.code %}selected{% endif %} >{{ _(language.name) }}</option>
+                    {% endfor %}
+                {% else %}
+                    {% for language in language_list %}
+                        <option value="{{ language.code }}" {% if default_language_code == language.code %}selected{% endif %} >{{ _(language.name) }}</option>
+                    {% endfor %}
```

Comment:
Would have been good to avoid this duplication here. Can we just pass the default code in `RealmCreationForm` as `initial_data`?